typescript/sdk/src/deploy-cctp-adapter.ts

```diff
@@ -0,0 +1,127 @@
+import { ethers } from 'ethers';
+import yargs from 'yargs';
+
+import { types, utils } from '@hyperlane-xyz/utils';
+
+import {
+  ChainMap, // ChainName,
+  Chains,
+  MultiProvider,
+  chainMetadata,
+} from '../src';
+
+import { hyperlaneContractAddresses } from './consts/environments';
+import {
+  AdapterType,
+  CctpAdapterConfig,
+  CctpAdapterDeployer,
+} from './middleware/liquidity-layer-v2/CctpAdapterDeployer';
+
+const circleDomainMapping = [
+  { hyperlaneDomain: chainMetadata[Chains.goerli].chainId, circleDomain: 0 },
+  { hyperlaneDomain: chainMetadata[Chains.fuji].chainId, circleDomain: 1 },
+  {
+    hyperlaneDomain: chainMetadata[Chains.arbitrumgoerli].chainId,
+    circleDomain: 3,
+  },
+];
+
+function buildCctpAdapterConfigMap(owner: types.Address) {
+  const config: ChainMap<CctpAdapterConfig> = {};
+  config[Chains.goerli] = {
+    type: AdapterType.CCTP,
+    tokenMessengerAddress: '0xd0c3da58f55358142b8d3e06c1c30c5c6114efe8',
+    token: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
+    tokenSymbol: 'USDC',
+    gasAmount: 300000,
+    circleDomainMapping: circleDomainMapping,
+    mailbox: hyperlaneContractAddresses.goerli.mailbox,
+    interchainGasPaymaster:
+      hyperlaneContractAddresses.goerli.interchainGasPaymaster,
+    interchainSecurityModule: '0x8EE099c620Fe635F86a4b97f3707359495022F37', // CctpIsm
```

Comment:
try avoiding hardcoding the addresses here

Reply:
moved to a config file